predicates/src/test/scala/com/wheaties/predicate/PredicateSpec.scala

```diff
@@ -61,6 +61,24 @@ class PredicateSpec extends WordSpecLike
       val filtered = sampleList.filter(Modulo(2, 0) or (Modulo(3, 0) and LessThen(6)))
       filtered should equal(List(2, 3, 4, 6))
     }
+
+    "have implicit `not` method" in new SpecExamples {
+//      val filtered = sampleList.filter(Predicate1.not.not(Modulo(2, 0)))
```

Comment:
Commented out line worked with old API. `Predicate.not.not` looked weird so I simply changed it to a method. Maybe I was missing something - how it was intended to be used?